docs/guide/developing.md

```diff
@@ -0,0 +1,14 @@
+# Developing with lit-html
```

Comment:
I was doubting whether this should be called `developing with lit-html` or `developing web components` ?

Reply:
renamed to just `Developing`